bson/src/main/org/bson/Vector.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson;
+
+
+import static org.bson.assertions.Assertions.isTrue;
+import static org.bson.assertions.Assertions.isTrueArgument;
+import static org.bson.assertions.Assertions.notNull;
+
+/**
+ * Represents a vector that is stored and retrieved using the BSON Binary Subtype 9 format.
+ * This class supports multiple vector {@link DataType}'s and provides static methods to create
+ * vectors.
+ * <p>
+ * Vectors are densely packed arrays of numbers, all the same type, which are stored efficiently
+ * in BSON using a binary format.
+ *
+ * @mongodb.server.release 6.0
+ * @see BsonBinary
+ * @since 5.3
+ */
+public abstract class Vector {
```

Comment:
Can't we just use the wording from `@Sealed` (`This class should be treated as sealed: it must not be extended or implemented by consumers of the library.`), instead of using quite a different wording that is currently present in the PR?

Reply:
I considered adding more context on the potential consequences; however, i am fine with keeping it consistent. I’ve updated it to use the `@Sealed` wording.